info/buildSrc/src/main/kotlin/org/cqfn/diktat/generation/docs/GenerationAvailableRules.kt

```diff
@@ -18,7 +18,7 @@ fun generateAvailableRules(rootDir: File, wpDir: File) {
         .map { it[2].replace("\\s+".toRegex(), "") to it[5] }
         .forEach { ruleMap[it.first]!!.config = it.second}
     val newText = File(wpDir, "sections/appendix.tex").readLines().toMutableList()
-    newText.removeAll(newText.subList(newText.indexOf("\\section*{available-rules}") + 1, newText.indexOf("\\section*{\\textbf{Diktat Coding Convention}}")))
+    newText.removeAll(newText.subList(newText.indexOf("\\section*{available-rules}") + 1, newText.indexOf("\\lstMakeShortInline[basicstyle=\\ttfamily\\bfseries]`")))
```

Comment:
I believe, there is still no section `available-rules`? Will `fromIndex` always be 0?

Reply:
Added new section